engine/runtime/src/main/scala/org/enso/compiler/pass/resolve/VectorLiterals.scala

```diff
@@ -76,7 +76,10 @@ case object VectorLiterals extends IRPass {
     copyOfIr: T
   ): T = copyOfIr
 
-  private def vectorCons(bindings: BindingsMap): IR.Expression = {
+  private def vectorModule(bindings: BindingsMap): IR.Expression = {
```

Comment:
This pass should be removed (also remove `IR.Error.Resolution.UnresolvedSequenceMacro`). Then change `SequenceLiteralNode` to just return a Vector and we're done.